### PR TITLE
Add Indy-Besu CI

### PR DESCRIPTION
**Pull request by Toktar**

### CODE REVIEW

**Comment by Toktar** [comment]
Could you move linters into separate jobs, please?
The build of artifacts can theoretically be used in parallel with smartcontracts unit tests

File: indy-besu/smart_contracts/package.json [hunk 1]
```diff
@@ -9,6 +9,9 @@
     "ts-lint": "tslint --fix -p tsconfig.json -c tslint.json",
     "solhint": "solhint --fix -f unix -c solhint.json contracts/**/*.sol",
     "lint": "yarn ts-lint && yarn solhint",
+    "ts-lint-check": "tslint -p tsconfig.json -c tslint.json",
+    "solhint-check": "solhint -c solhint.json contracts/**/*.sol",
+    "lint-check": "npm run ts-lint-check && npm run solhint-check",
```
Comment (akarabashov) [inline indy-besu/smart_contracts/package.json]: Why did we use npm instead of yarn here?

File: .github/workflows/indy-besu.yml
Comment (akarabashov) [inline .github/workflows/indy-besu.yml:1]: Maybe we should add running demo scripts?

File: indy-besu/smart_contracts/package.json (hunk 1 above)
Comment (DenisRybas) [reply indy-besu/smart_contracts/package.json]: Changed to yarn

File: .github/workflows/indy-besu.yml
Comment (DenisRybas) [reply .github/workflows/indy-besu.yml:1]: added

**Review by akarabashov — APPROVED** [review]

**Pull request merged**